src/auth/jwtOptions.ts

```diff
@@ -0,0 +1,32 @@
+import { expressJwtSecret } from 'jwks-rsa';
+import { requireEnv } from 'require-env-variable';
+import type {
+  GetVerificationKey,
+  Params,
+} from 'express-jwt';
+
+const {
+  AUTH_SERVER_ISSUER,
+} = requireEnv(
+  'AUTH_SERVER_ISSUER',
+);
+
+// The `jsonwebtoken` type `VerifyOptions` can have issuer as either `string` or `string[]`. In
+// test code, we would like to use the valid exact issuer that was set up here. Therefore export
+// issuer separately as a string and re-use below when setting jwtOptions.
+export const issuer = AUTH_SERVER_ISSUER;
+
+export const jwtOptions: Params = {
+  // Because of different types from version-to-version in express-jwt (see jwks-rsa notes), cast
+  // to the later express-jwt API's GetVerificationKey.The jwks-rsa notes can be found in
+  // `node_modules/jwks-rsa/index.d.ts` or here:
+  // https://github.com/auth0/node-jwks-rsa/blob/83d0202a9f462451c0e1e2adb1d91280ee1ef30c/index.d.ts#L60
+  secret: expressJwtSecret({
+    cache: false,
+    rateLimit: false,
```

Comment:
Instead, I will remove these lines.

Reply:
Thanks!

Looks like the defaults are to enable the cache and disable the rate limit ([source](https://github.com/auth0/node-jwks-rsa/blob/59373c5cd5b357fbb56b621c943f002e9f3076ca/src/JwksClient.js#L7-L14)). I think we want the rate limit on, too, to prevent a DDoS attack from spreading from the service to the IdP - right?

Reply:
I recall hitting rate limits when I had `mock-jwks` set up and torn down multiple times during tests (if I recall correctly, even with only one `mock-jwks` per test suite/file). Now that we use one `mock-jwks` and one mock bearer JWT, I doubt that issue would occur with rate limits set. I will set the rate limit low during tests to see if it is exceeded (I do not expect it to, even at `1`) to double-check it is working the way I expect, and then set it to high if that works fine.

Reply:
Ten per minute seemed low to me. So I set it to ten times that.